### ISSUE TITLE
Fix banner image for start page

### ISSUE THREAD

**Issue by marcoow** [mainmatter/mainmatter.com#13]
Dimensions are too small (this is on iPad)
![image](https://cloud.githubusercontent.com/assets/1510/15629626/302c22ba-251e-11e6-9154-b46435b0ddc1.png)

**Comment by marcoow** [comment]
The best option to fix this is probable to use an SVG background image: http://caniuse.com/#feat=svg-css

[logo.svg.zip](https://github.com/simplabs/simplabs.github.io/files/316501/logo.svg.zip)

**Comment by ganeshbsub** [comment]
@marcoow Did you write the vector code for the logo yourself or did you use some conversion tool?

**Comment by marcoow** [comment]
@ganeshbsub: that's just the EPS exported as SVG with [Affinity Designer](https://affinity.serif.com/).

**Comment by ganeshbsub** [comment]
That's pretty expensive. I tried to create a vector image with Photoshop, but that doesn't work. I exported the file as SVG and all it does is embed the image as a layer in the SVG XML and does not actually create vector data. I'll try to create a true SVG manually, but it looks like changing the `background-size` to 100% 100% instead of `contain` makes the image scale properly for now. See #22. The previous fix from #15 doesn't seem to work.
